table/queries/merged_reduced_scans.sql

```diff
@@ -84,23 +84,23 @@ CREATE TEMP FUNCTION ClassifyError(error STRING,
     WHEN REGEXP_CONTAINS(error, "malformed MIME") THEN "http/http.invalid"
 
     # Content verification failures
-    WHEN (source = "ECHO" AND ENDS_WITH(error, "EOF")) THEN "content/response_mismatch" # Echo
+    WHEN (source = "ECHO" AND ENDS_WITH(error, "EOF")) THEN "content/unexpected" # Echo
     # Hyperquack v1 errors
-    WHEN (error = "Incorrect echo response") THEN "content/response_mismatch" # Echo
-    WHEN (error = "Received response") THEN "content/response_mismatch" # Discard
-    WHEN (error = "Incorrect web response: status lines don't match") THEN "content/status_mismatch" # HTTP/S
-    WHEN (error = "Incorrect web response: bodies don't match") THEN "content/body_mismatch" # HTTP/S
-    WHEN (error = "Incorrect web response: certificates don't match") THEN "content/tls_mismatch" # HTTPS
-    WHEN (error = "Incorrect web response: cipher suites don't match") THEN "content/tls_mismatch" # HTTPS
-    WHEN (error = "Incorrect web response: TLS versions don't match") THEN "content/tls_mismatch" # HTTPS
+    WHEN (error = "Incorrect echo response") THEN "content/unexpected" # Echo
+    WHEN (error = "Received response") THEN "content/unexpected" # Discard
+    WHEN (error = "Incorrect web response: status lines don't match") THEN "content/unexpected_status" # HTTP/S
+    WHEN (error = "Incorrect web response: bodies don't match") THEN "content/unexpected_body" # HTTP/S
+    WHEN (error = "Incorrect web response: certificates don't match") THEN "content/unexpected_tls" # HTTPS
```

Comment:
I'd actually differentiate the TLS cases: `certificate_mismatch`, `cipher_mismatch`, `tls_version_mismatch`.

That may help us verify consistency and find MITM cases.

Reply:
These three together are [>.1% of all https outcomes](https://docs.google.com/spreadsheets/d/1HisIsJWbsiWjp9sI8cU7xTpVao2L1oB4chiNKykclWM/edit?resourcekey=0-azZzA3FIPEODKRWnp-hpsA#gid=655449305) which is why I had combined them. I could be convinced splitting is a good idea, but I'd like to see an actual case in the dashboard where it helps analyse something. I don't think it's related to the change I'm trying to make in this PR.

Reply:
I agree it doesn't have to be part of this PR. I guess if we start seeing TLS issues we can add those cases, but I'd rather have more cases initially and simplify as we gain understanding. The low number of cases means it won't affect most usage, but it may help finding MITM when it actually happens.

Not high priority. It may be worth investigating some of the existing TLS errors at some point. I wonder if we can see the Kazakhstan MITM.

Reply:
The Kazakhstan MITM isn't in the main dataset unfortunately.